english/love/love.md

```diff
@@ -13,4 +13,6 @@ out of love, make the best of now while being aware of the worst of before.
 Of course, there is no substitute for the experince of it. It must be
 lived in order to be understood. All the words there are in the world 
 cannot make you understand the feeling of it. Anyways, [here's 
-something comepletely different.](../marshmallow.md)
\ No newline at end of file
+something comepletely different.](../marshmallow.md)
+
+If all else fails, you could [try this.](english/read-a-book/read-a-book.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.